index.html

```diff
@@ -1578,8 +1579,8 @@ <h3>Issuer</h3>
           <dt><var>issuer</var></dt>
           <dd>
 The value of the <code>issuer</code> <a>property</a> MUST be either a
-<a>URI</a> or an object containing an <code>id</code> <a>property</a>. It is
-RECOMMENDED that the <a>URI</a> in the <code>issuer</code> or its
+<a>URL</a> or an object containing an <code>id</code> <a>property</a>. It is
```

Comment:
are DIDs URLs?

Reply:
Yes. I believe we checked that in the lifetime of the DID WG.

Reply:
A [DID is a URI](https://www.w3.org/TR/did-core/#did-syntax), which [is a URL per the WHATWG spec](https://url.spec.whatwg.org/#goals). Additionally, every DID is also a DID URL, [per the syntax rules in the DID core spec](https://www.w3.org/TR/did-core/#did-url-syntax). Note a DID with an empty path, no optional query, and no optional fragment is both a DID and a DID URL.

Reply:
@OR13 asked:
> are DIDs URLs?

Yes, they are. (citation above)